packages/@expo/cli/src/start/server/DevToolsPluginManager.ts

```diff
@@ -27,7 +55,61 @@ export default class DevToolsPluginManager {
     }
     const plugins = (await this.queryAutolinkedPluginsAsync(this.projectRoot)).map((plugin) => ({
       ...plugin,
-      webpageEndpoint: `${DevToolsPluginEndpoint}/${plugin.packageName}`,
+      webpageEndpoint: plugin.webpageRoot
+        ? `${DevToolsPluginEndpoint}/${plugin.packageName}`
+        : undefined,
+      decsription: plugin.cli?.description ?? '',
+      executor: plugin.cli?.main
+        ? async ({ command, args, apps }: DevToolsPluginCliExecutorArguments) => {
+            return new Promise<string>(async (resolve, reject) => {
+              // Set up the command and its arguments
+              const tool = path.join(plugin.packageRoot, plugin.cli!.main);
+              const child = spawn(
+                'node',
+                [tool, command, `'${JSON.stringify(args)}'`, `'${JSON.stringify(apps)}'`],
+                {
+                  cwd: this.projectRoot,
+                  shell: true,
+                  env: { ...process.env, FORCE_COLOR: '1' },
+                }
+              );
```

Comment:
not sure whether we can show a hint in cli menu that menu will execute script from `tool` (e.g. `/app/node_modules/pkg/cli.js`)

Reply:
This is so that if the tool outputs data with formatted contents to stdout it will use this formatting info.